ci/builders/linux_host_engine.json

```diff
@@ -78,7 +78,6 @@
                         "out/host_debug/zip_archives/linux-x64/font-subset.zip",
                         "out/host_debug/zip_archives/flutter_patched_sdk.zip",
                         "out/host_debug/zip_archives/dart-sdk-linux-x64.zip",
-                        "out/host_debug/zip_archives/flutter-web-sdk-linux-x64.zip",
```

Comment:
Does this mean we don't need this artifact anymore?

Reply:
That's correct. The flutter tool no longer uses any of the host specific flutter-web-sdk artifacts (i.e. `flutter-web-sdk-{platform-name}.zip`). It now just uses `flutter-web-sdk.zip` which is host-agnostic, and that is produced by the linux_host_engine builder.

Reply:
Thanks for the explanation. LGTM.

Reply:
Thank you!